### ISSUE TITLE
The ground at the water's edge is not visible through the water at the graphics settings “Between Low and Mid”, “Mid”, “Between Mid and High”

### ISSUE THREAD

**Issue by katerynamproductengine** [secondlife/viewer#2720]
### Environment

<details><summary>Environment:</summary>
Second Life Release 7.1.11.10947335521 (64bit)
Release Notes

You are at 225.9, 67.0, 22.6 in Bug Island 2 located at simhost-05184521a8887a5b8.agni
SLURL: http://maps.secondlife.com/secondlife/Bug%20Island%202/226/67/23
(global coordinates 260578.0, 249411.0, 22.6)
Second Life Server 2024-07-22.10048683488
Release Notes

CPU: Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz (1600 MHz)
Memory: 16384 MB
OS Version: macOS 13.6.9 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Mon Jun 24 01:25:37 PDT 2024; root:xnu-8796.141.3.706.2~1/RELEASE_X86_64 x86_64
Graphics Card Vendor: Intel Inc.
Graphics Card: Intel(R) UHD Graphics 617

OpenGL Version: 4.1 INTEL-20.7.2

Window size: 1440x876
Font Size Adjustment: 96pt
UI Scaling: 1
Draw distance: 96m
Bandwidth: 3000kbit/s
LOD factor: 1.125
Render quality: 1
Texture memory: 1536MB
Disk cache: Max size 1638.4 MB (0.2% used)
HiDPI display mode: 

J2C Decoder Version: KDU v7.10.4
Audio Driver Version: OpenAL, version 1.1 ALSOFT 1.23.1 / OpenAL Community / OpenAL Soft: OpenAL Soft
Dullahan: 1.14.0.202408091638
  CEF: 118.4.1+g3dd6078+chromium-118.0.5993.54
  Chromium: 118.0.5993.54
LibVLC Version: 3.0.21
Voice Server Version: Vivox 4.10.0000.32327.5fc3fe7c.5942f08

Packets Lost: 12/2431 (0.5%)
September 27 2024 06:35:21
</details>

### Description

See reproduction steps

### Reproduction steps

1. Log into the viewer on Agni grid at the location: http://maps.secondlife.com/secondlife/Bug%20Island%202/226/67/23 facing South. 
2. Set graphics settings from “Low” to “Between Low and Mid “, “Mid” or “Between Mid and High”.
3. Observe the environment.

**Actual result:** The ground at the water's edge is not visible through the water at the graphics settings “Between Low and Mid”, “Mid”, “Between Mid and High”.

Actual screenshots:
<details><summary>Between Low and Mid:</summary>
<img width="1440" alt="Screenshot 2024-09-27 at 2 35 58 PM" src="https://github.com/user-attachments/assets/9946d4fc-d561-4d43-973f-92507d43ba4f">
</details>
<details><summary>Mid:</summary>
<img width="1440" alt="Screenshot 2024-09-27 at 2 36 10 PM" src="https://github.com/user-attachments/assets/ba3dae43-4bb6-4327-8148-43ae55f9ccc2">
</details>
<details><summary>Between Mid and High:</summary>
<img width="1440" alt="Screenshot 2024-09-27 at 2 39 38 PM" src="https://github.com/user-attachments/assets/91a8289a-505c-4599-bb6c-0bf8ca6ba0e9">
</details>

**Expected result:** The ground at the water's edge is visible through the water at the graphics settings “Between Low and Mid”, “Mid”, “Between Mid and High”.

**Additional information:**
This issue was found in the scope of [IQA #353](https://github.com/secondlife/iqa/issues/353) during Environment Rendering conformity test -> Sky and water tests -> Test 3/ Test4/ Test5 (https://lindenlab.atlassian.net/wiki/x/oIGaL).
This issue is reproduced on Win10/OSX.
This issue is reproduced on the current release 7.1.10.10800445603 build.

**Comment by AtlasLinden** [comment]
TLDR 
Pre-PBR: Transparent Water setting is enabled at Low+ and above 
ExtraFPS: Transparent Water setting is enabled at High and above
